### PR TITLE
Fix ineffassign issues

### PR DESCRIPTION
**Pull request by zhouhao3**

### CODE REVIEW

**Comment by rjbez17** [comment]
@zhouhao3 thanks for this, can you please squash your commits?

/ok-to-test
/assign @JimBugwadia

**Comment by zhouhao3** [comment]
updated, PTAL.

File: incubator/virtualcluster/cmd/kubectl-vc/util.go [hunk 1]
```diff
@@ -94,6 +94,9 @@ func readFromFileOrURL(path string) ([]byte, error) {
 		defer resp.Body.Close()
 
 		yamlContent, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return nil, err
+		}
```
Comment (christopherhein) [inline incubator/virtualcluster/cmd/kubectl-vc/util.go:99]: This code base was moved to sigs.k8s.io/cluster-api-provider-nested/virtualcluster/cmd/kubectl-vc/util.go I would probably remove this file change as to not cause issues later when we finally delete that directory from this repo. 

/cc @Fei-Guo

Comment (Fei-Guo) [reply incubator/virtualcluster/cmd/kubectl-vc/util.go:99]: Yes, agree. You can submit a PR to cluster-api-provider-nested/virtualcluster repo instead.

**Comment by zhouhao3** [comment]
@christopherhein @Fei-Guo updated, PTAL

**Comment by christopherhein** [comment]
@zhouhao3 since this work was moved to CAPN, can we close this PR?

**Comment by zhouhao3** [comment]
yes, done.

**Pull request closed without merging**